uint16 detect_word
uint16 send_goal
uint16 detect_object
float32[] object_position
float32[] start_arm
bool finish_arm
bool go_back
